js/index.js

```diff
@@ -12,15 +12,118 @@ const skillsList = ['C++', 'Revit', 'AutoCAD', 'Microsoft Office'];
 const skillsSection = document.getElementById('Skills');
 const skillUL = skillsSection.querySelector('ul');
 
-for (skill of skillsList){
-   let skillItem = document.createElement('li');
-   skillItem.innerHTML = skill;
-   skillItem.style.fontSize = '1.4em';
-    skillUL.appendChild(skillItem);
+for (let skill of skillsList) {
+  let skillItem = document.createElement('li');
+  skillItem.innerHTML = skill;
+  skillItem.style.fontSize = '1.4em';
+  skillUL.appendChild(skillItem);
 }
 
 document.querySelectorAll('.experience-item').forEach(item => {
-    item.addEventListener('click', () => {
-      item.classList.toggle('open');
+  item.addEventListener('click', () => {
+    item.classList.toggle('open');
+  });
+});
+
+
+//handle events for the message form
+//
+let messageForm = document.querySelector("[name='leave_message']");
+let messageSection = document.getElementById('message-section');
+let messageList = messageSection.querySelector('ul');
+messageSection.hidden = true;
+
+let idCpunter = 0;
```

Comment:
You have a typo here.
```suggestion
let idCounter = 0;
```